config/values.yml

```diff
@@ -4,6 +4,8 @@ namespace: kapp-controller
 packaging_global_namespace: kapp-controller-packaging-global
 create_namespace: true
 create_packaging_namespace: true
+#! clusters version 1.19 and below should disable APIPriorityAndFairness
+enable_APIPriorityAndFairness: true
```

Comment:
is it weird casing for flag/values? should we have more general configuration for gates? eg `feature_gates: ["APIPriorityAndFairness=true"]` or somethign like that?

Reply:
It probably would be a better long term decision to make it more general to feature gates. I wouldn't be opposed to changing the format.

Reply:
@cppforlife given we're using this flags lib currently https://pkg.go.dev/flag which doesn't support lists it seems like we'd have to do some bespoke string scraping,  doesn't feel totally warranted to set up for N=1 feature flag.   I do agree though that we're missing some feature-flag machinery and also what is up with the mixed camel/snake case variable (shouldn't that be a compiler error?).

Reply:
@cppforlife we dug into a bit further and continue to feel like 
- there's only one flag currently
- to make it general is a non-trivial expansion of this PR (either we're parsing strings or we're passing unchecked user input directly to the k8s api, so we're parsing strings which is exactly what the flags parser is for already, but it doesn't support lists so we're back to using a single scalar flag.)
- the flag is only for the sake of k8s versions that are already out of their support window
- likely by the time we add a second flag, we'll be ready to remove this flag entirely (i.e. next year when we bump k8s lib versions again, we can entirely drop 1.19 support)